rasa/nlu/extractors/extractor.py

```diff
@@ -469,3 +469,19 @@ def check_correct_entity_annotations(training_data: TrainingData) -> None:
                         docs=DOCS_URL_TRAINING_DATA_NLU,
                     )
                     break
+
+
+class EntityExtractorMixin(EntityExtractor):
+    """Class for backward compatibility in 3.x.
+
+    EntityExtractorMixin was renamed to EntityExtractor in 3.3.
+    This class will be removed in 4.0
+    """
+
+    def __init_subclass__(cls, **kwargs):
```

Comment:
Code Quality check failed because the parameters in this method need to have type hints. Please add them, kwargs can be of type `Any`.